### PR TITLE
LIBCLOUD-505: Added four new calls to support pause/unpause and suspend/...

### PR DESCRIPTION
**Pull request by cderamus**
...resume calls. These calls are supported in the v1.1 of OpenStack and are useful for situations such as performing system maintenance.

### CODE REVIEW

File: libcloud/compute/drivers/openstack.py [hunk 1]
```diff
@@ -2092,6 +2092,30 @@ def ex_get_metadata_for_node(self, node):
         """
         return node.extra['metadata']
 
+    def ex_pause_node(self, node):
+        uri = '/servers/%s/action' % (node.id)
+        data = {'pause': None}
```
Comment (Kami) [inline libcloud/compute/drivers/openstack.py:2097]: (I know it's not your fault, I'm more or less just complaining)

That's a weird API.

**Comment by Kami** [comment]
In any case, the patch looks good so I've merged it into trunk.

Thanks!

File: libcloud/compute/drivers/openstack.py (hunk 1 above)
Comment (cderamus) [inline libcloud/compute/drivers/openstack.py:2097]: I couldn't agree more :).

**Comment by cderamus** [comment]
Thanks!

**Pull request closed without merging**